students/etwum/lesson03/strformat_lab.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+
+# Task One
+
+x = (2, 123.4567, 10000, 12345.67)
+
+file = "file{0:03d}: {1:.2f}, {2:.2e}, {3:.2e}".format(x[0],x[1],x[2],x[3])
+print(file)
+print("----------------------------------------------------------------")
+
+# Task Two
+
+x1 = x[0]
+x2 = x[1]
+x3 = x[2]
+x4 = x[3]
+
+string = f"file{x1:03d}: {x2:.2f}, {x3:.2e}, {x4:.2e}"
+print(string)
+print("----------------------------------------------------------------")
+
+# Task Three
+
+def formatter(*tuple):
+    # creates a string which dynamically prints a string based on the numbers passed to the function
+    form_string = "the " + str(len(tuple)) + " numbers are " + ", ".join(["{}"]*len(tuple)).format(*tuple)
+    return print(form_string)
```

Comment:
`print()` function doesn't actually return anything, so either you should have `return = "the " ... ` or remove `return`